go-controller/pkg/ovn/base_network_controller_pods.go

```diff
@@ -1177,3 +1190,21 @@ func (bnc *BaseNetworkController) wasPodReleasedBeforeStartup(uid, nad string) b
 	}
 	return bnc.releasedPodsBeforeStartup[nad].Has(uid)
 }
+
+func checkBridgeMapping(physicalNetworkName string) error {
+	ovnBridgeMappings, stderr, err := util.RunOVSVsctl("--if-exists", "get", "Open_vSwitch", ".",
+		"external_ids:ovn-bridge-mappings")
+	if err != nil {
+		return fmt.Errorf("failed to get ovn-bridge-mappings from the OVS global table. stderr: %s (%v)", stderr, err)
+	}
+
+	bridgeMappings := strings.Split(ovnBridgeMappings, ",")
+	for _, bridgeMapping := range bridgeMappings {
+		m := strings.Split(bridgeMapping, ":")
+		if m[0] == physicalNetworkName {
+			return nil
+		}
+	}
+
+	return fmt.Errorf("failed to find bridge mapping for physicalNetworkName: %q, ovn-bridge-mappings external_ids %s", physicalNetworkName, ovnBridgeMappings)
```

Comment:
half asking: bridge mapping is node specific, shouldn't we add on what node is the bridge-mapping absent?

Reply:
the error will be presented on a pod, and the pod is node specific already

Reply:
right, this is the msg that will be listed on pod's describe operation, right ? 

If so, I agree: no point logging the node name there. 

... it would be totally different if this were a log on ovnkube-controller ...

Reply:
yeah this is going to be on the pod events so it's easy to know what is the node.
But keeping in mind the story of "user created a bridge-mapping on desired node, user created vm, user trying to figure out why the vm is not running" it would help to shed a light in the event that the pod got to the wrong place. 
But it's not a must indeed - up to you..

Reply:
... I mean ... it doesn't cost much to add the node name tbh ...